litmus-portal/graphql-server/graph/schema.resolvers.go

```diff
@@ -76,6 +78,14 @@ func (r *mutationResolver) PodLog(ctx context.Context, log model.PodLog) (string
 	return mutations.LogsHandler(log, *store)
 }
 
+func (r *mutationResolver) AddMyHub(ctx context.Context, myhubInput model.CreateMyHub, username string) (*model.User, error) {
+	return myhub.AddMyHub(ctx, myhubInput, username)
+}
+
+func (r *mutationResolver) UpdateMyHub(ctx context.Context, username string, updateMyHub model.UpdateMyHub) (*model.User, error) {
+	panic(fmt.Errorf("not implemented"))
```

Comment:
Why this function is not implemented yet? If not you can remove it from gql schema

Reply:
Sure @rajdas98 , this can be removed since we don't have any option to update or delete a hub.